src/components/molecules/GlobalNavigation.jsx

```diff
@@ -3,6 +3,7 @@ import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import { em } from 'polished';
 import windowSize from 'react-window-size';
+import SmoothScroll from 'smooth-scroll/dist/smooth-scroll'
```

Comment:
❓ Couldn't you import directly from the npm package name?

```suggestion
import SmoothScroll from 'smooth-scroll';
```

Also, missing a semi-colon. With the right plugins and settings you can setup auto-fix on save so that most of these common misses are auto-fixed.  

**VS Code Extensions**
- ESLint (2.1.1)
- Prettier (3.20.0)

**Settings:**
```
"editor.formatOnSave": true,
"editor.codeActionsOnSave": {
    "source.fixAll.eslint": true
  },
```

Reply:
The import from the dist is because the package references window, so when Gatsby imports it to complete a build, it crashes.  This skips the package's wrapper that references window.  If there's a more "gatsby"-like solution, I'd be very happy to hear about it.

Reply:
@mewdriller do you know of a more gatsby-like approach to this window issue that Zach ran into?

Reply:
https://www.gatsbyjs.org/docs/using-client-side-only-packages/

They don't have anything specific, and their first recommendation is to "work around the problem"